torch/_logging/_internal.py

```diff
@@ -227,6 +227,7 @@ def set_logs(
     cudagraphs: bool = False,
     sym_node: bool = False,
     compiled_autograd_verbose: bool = False,
+    ts2ep_conversion: bool = False,
```

Comment:
I don't think we need to add this specific flag to TORCH_LOGS.. having it under TORCH_LOGS="+export" seems good enough to me